invite/templates/invite/reset_email.txt

```diff
@@ -8,4 +8,6 @@ Your username is: {{ username }}
 Your password can be reset at this unique page. Do not share this URL with anyone.
 http://{{ domain }}{% url 'invite:reset' %}?reset_code={{ reset_code }}
 
+Please note, your activation link is valid only until tomorrow.
```

Comment:
So I think this could be interpreted as the link only being valid for today, and then invalid starting tomorrow. For clarity, maybe we could change this to "Please note, your activation link will expire after tomorrow."?

Reply:
or `Please note, your activation link is valid through tomorrow.`

Reply:
heh, sorry for being a bit late on that. I think what's there currently is still acceptable